gurmukhiutils/unicode.py

```diff
@@ -0,0 +1,448 @@
+from typing import Literal, Match
+
+UNICODE_STANDARDS = Literal["Unicode Consortium", "Sant Lipi"]
+
+
+def unicode(
+    string: str,
+    unicode_standard: Literal[UNICODE_STANDARDS] = "Unicode Consortium",
+) -> str:
+    """
+    Converts any ascii gurmukhi characters and sanitizes to unicode gurmukhi.
+
+    Args:
+        string: The string to affect.
+
+        unicode_standard: The mapping system to use. The default is unicode compliant and can render 99% of the Shabad OS Database. The other option "Sant Lipi" is intended for a custom unicode font bearing the same name (see: https://github.com/shabados/SantLipi). Defaults to "Unicode Consortium".
+
+    Returns:
+        A string whose Gurmukhi is normalized to a Unicode standard.
+
+    Examples:
+        >>> unicode("123")
+        '੧੨੩'
+
+        >>> unicode("<> > <")
+        'ੴ ☬ ੴ'
+
+        >>> unicode("gurU")
+        'ਗੁਰੂ'
+    """
+
+    import re
+
+    # AnmolLipi/GurbaniAkhar & GurbaniLipi by Kulbir S. Thind, MD
+    common_ascii_to_unicode_map = {
```

Comment:
Constants are usually defined to be all caps
https://peps.python.org/pep-0008/#constants

You might want something like an encoding dir with unicode.py and constants.py and do something like
from gurmukhiutils.encoding import unicode, constants